src/components/Quiz/QuizItem.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useContext } from "react"
+import { Box, Flex, ListItem, Stack, Text } from "@chakra-ui/react"
+import { useTranslation } from "gatsby-plugin-react-i18next"
+
+import Button from "../Button"
+import Translation from "../Translation"
+import Tag from "../Tag"
+import { GreenTickIcon } from "../icons/quiz"
+
+import { QuizzesHubContext } from "./context"
+
+import { trackCustomEvent } from "../../utils/matomo"
+
+import { QuizzesListItem } from "../../types"
+
+import allQuizzesData from "../../data/quizzes"
+
+const QuizItem: React.FC<QuizzesListItem> = (props) => {
+  const {
+    num,
+    title,
+    id,
+    level,
+    next,
+    quizHandler,
+    nextHandler,
+    modalHandler,
+  } = props
+  const { completed } = useContext(QuizzesHubContext)
+  const numberOfQuestions = allQuizzesData[id].questions.length
+  const isCompleted = JSON.parse(completed)[id][0]
+
+  const { t } = useTranslation()
+
+  const handleStart = () => {
+    quizHandler(id)
+    nextHandler(next)
+    modalHandler(true)
+
+    trackCustomEvent({
+      eventCategory: "quiz_hub_events",
+      eventAction: "quizzes click",
+      eventName: `${id}`,
+    })
+  }
+
+  return (
+    <ListItem
+      color={isCompleted ? "bodyMedium" : "text"}
+      fontWeight="bold"
+      px={{ base: 0, lg: 4 }}
+      py={4}
+      borderBottom="1px solid"
+      borderColor="disabled"
+      mb={0}
+    >
+      <Flex
+        justifyContent="space-between"
+        alignItems={{ base: "flex-start", lg: "center" }}
+        direction={{ base: "column", lg: "row" }}
+      >
+        <Stack mb={{ base: 5, lg: 0 }}>
+          <Flex gap={2} alignItems="center">
+            <Text
+              color={isCompleted ? "bodyMedium" : "text"}
+              fontWeight="bold"
+              mb={0}
+            >
+              {`${num}. ${title}`}
```

Comment:
```suggestion
              {`${num}. ${t(id)}`}
```

We should use the page ID to pull the translated string if available; most of them we already have from `common.json` as they are the same as the nav text

<img width="522" alt="Screenshot 2023-06-02 at 12 58 11" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/8ce236bb-7c71-4862-9afa-1c5649726a36">

Reply:
Another approach might be to move the list counting out of the text content and use the `<Translation>` component instead.

Currently, we hide the list style, then add it in manually using the index + 1

```js
{`${num}. ${id}`}
```

The problem is that this is read twice on some screenreaders: e.g. "One, one, What is Ethereum"

We could use a counter on the Text wrapper

```js
  _before={{
    content: 'counter(list-counter) ". "',
  }} 
```

So QuizItem would be like this:

```js
// QuizItem.tsx
<ListItem
  color={isCompleted ? "bodyMedium" : "text"}
  fontWeight="bold"
  px={{ base: 0, lg: 4 }}
  py={4}
  borderBottom="1px solid"
  borderColor="disabled"
  mb={0}
  sx={{ counterIncrement: "list-counter" }}
>

...

<Text
  color={isCompleted ? "bodyMedium" : "text"}
  fontWeight="bold"
  mb={0}
  _before={{
    content: 'counter(list-counter) ". "',
  }}    
>
  <Translation id="id" />
</Text>
```

And add a reset to Quizzes list

```js
//QuizzesList
 <OrderedList m={0} listStyleType="none" sx={{ counterReset: "list-counter" }}>
```

Reply:
@minimalsm the item count is a hack, as it's hard to implement the current layout design, as item marker has to be 'inside' and next to title, which breaks the layout of the flex component (you can test it)

Reply:
Ooh, nice one, much better than current solution, thx! Fixed now